wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/networking/PocketCastsNetworkingRules.kt

```diff
@@ -55,27 +52,22 @@ object PocketCastsNetworkingRules : NetworkingRules {
         networks: Networks,
         requestType: RequestType,
     ): NetworkStatus? {
-
-        // Always prefer Wifi if it is active
-        networks.networks.firstOrNull { it.networkInfo is NetworkInfo.Wifi }?.let {
-            return it
-        }
-
-        return when (requestType) {
-            DownloadRequest -> {
-                // For downloads force LTE as the backup to Wifi, to avoid slow downloads.
-                networks.networks.firstOrNull {
-                    it.networkInfo.type == NetworkType.Cell
-                }
```

Comment:
This was the problem before. If there was not a cell network it was returning null, which meant that the call [would fail](https://github.com/google/horologist/blob/bb2deb64dd3bdcd8eb8ef291c1339819ef81a558/network-awareness/okhttp/src/main/java/com/google/android/horologist/networks/okhttp/NetworkSelectingCallFactory.kt#L94-L101).